pyproject.toml

```
@@ -14,6 +14,10 @@ authors = [
   {email = "reiter.christoph@gmail.com"},
   {name = "Christoph Reiter"}
 ]
+dependencies = [
+    # for @deprecated decorator
+    'typing_extensions>="4.5.0"; python_version<"3.13"',
```

Sorry, this should actually be:

```suggestion
    "typing_extensions>=4.5.0; python_version<'3.13'",
```

The syntax I previously suggested will fail with the following:

```
ValueError: invalid pyproject.toml config: `project.dependencies[0]`.
configuration error: `project.dependencies[0]` must be pep508
```